RPIPs/RPIP-42.md

```diff
@@ -25,6 +25,14 @@ This proposal also explicitly tries to benefit the smallest NOs in a few ways, i
 - We give precedence to small nodes staking some number of initial validators in the Node Operator queue (see [RPIP-59](RPIP-59))
 - There is a small but tangible financial benefit for large stakers that stake as few large nodes instead of many small nodes -- this (alongside our vote power, which scales with the square root of vote-eligible RPL) helps preserve the strong governance voice of small NOs
 
+## Motivation
+
+The primary motivation for the introduction of smaller ETH bonds in this form is twofold. First, lower ETH bonds allow for the same amount of bonded ETH to support a larger volume of rETH, this helps unlock growth for the protocol, as it has been primarily limited by lack of Node Operators in the months preceding this proposal. Second, lower ETH bonds increase the profitability for Node Operators, allowing the protocol to grow and compete favorably with alternative liquid staking providers.
```

Comment:
I'd tweak this a bit. Since we have UARs, it need not be NOs that get the additional profit. Rather, the overall system is more profitable, and then we steer it where needed (which could be NOs).
> Second, lower ETH bonds increase the capital efficiency of the protocol. The improved return per total investment allows the protocol to better compete with alternative liquid staking providers, eg, by paying node operators more.

Reply:
Updated.